### PR TITLE
Misc Fixes

### PR DESCRIPTION
**Pull request by jarveson**
Some edat fixes, fixes Super rub a dub booting and, should mostly take care of #1288. 
Small speed increase on opening settings from sprx files
Fixes gameContentDialog freeze, (would pop up with white text and hard freeze emulator)
Fixes cellPadGetData, fixes DMC analog sticks and #1923

### CODE REVIEW

File: rpcs3/Emu/PSP2/Modules/sceNpManager.cpp [hunk 1]
```diff
@@ -60,14 +60,14 @@ s32 sceNpManagerGetChatRestrictionFlag(vm::ptr<s32> isRestricted)
...
-	REG_FUNC(0x19E40AE1, sceNpTerm);
-	REG_FUNC(0x3C94B4B4, sceNpManagerGetNpId);
-	REG_FUNC(0x54060DF6, sceNpGetServiceState);
-	REG_FUNC(0x44239C35, sceNpRegisterServiceStateCallback);
-	REG_FUNC(0xD9E6E56C, sceNpUnregisterServiceStateCallback);
-	REG_FUNC(0x3B0AE9A9, sceNpCheckCallback);
-	REG_FUNC(0xFE835967, sceNpManagerGetAccountRegion);
-	REG_FUNC(0xAF0073B2, sceNpManagerGetContentRatingFlag);
-	REG_FUNC(0x60C575B1, sceNpManagerGetChatRestrictionFlag);
+	REG_FUNC(0x04D9F484, _sceNpInit);
```
Comment (Nekotekina) [inline rpcs3/Emu/PSP2/Modules/sceNpManager.cpp]: Revert this. In the case of problems add thread& arg.

**Comment by SakataGintokiYT** [comment]
Fotoszok.pl
add this too : 
s32 sceNpBasicGetFriendListEntry()
  {	
 	sceNp.warning("sceNpBasicGetFriendListEntry()");
 	return SCE_NP_ERROR_OFFLINE;
  }

File: rpcs3/Emu/PSP2/Modules/sceNpManager.cpp (hunk 1 above)
Comment (jarveson) [inline rpcs3/Emu/PSP2/Modules/sceNpManager.cpp]: Sure, ill just let scribam take care of the scenp stuff in his new pr

File: rpcs3/Crypto/unself.cpp [hunk 2]
```diff
@@ -1109,8 +1104,11 @@ bool SELFDecrypter::DecryptNPDRM(u8 *metadata, u32 metadata_size)
 	}
 	else if (ctrl->npdrm.license == 3)  // Free license.
 	{
-		// Use the NP_KLIC_FREE.
-		memcpy(npdrm_key, NP_KLIC_FREE, 0x10);
+		// Use klicensee if available.
```
Comment (Nekotekina) [inline rpcs3/Crypto/unself.cpp]: Why it's only for free license?

Comment (jarveson) [inline rpcs3/Crypto/unself.cpp]: Its the only spot where it seems to actually be used. Theres cases where the edat has local license flag, and npDrmIsAvailable() is called with a klicense_key, but in that case it needs to be ignored and the rap is used instead. All the cases I've found that actually use the klicense given are flagged with 'free-license'

**Pull request merged**